openbb_terminal/core/models/credentials_model.py

```diff
@@ -76,3 +76,14 @@ def __repr__(self) -> str:
             dataclass_repr += f"    {key}='{value}', \n"
 
         return f"{self.__class__.__name__}(\n{dataclass_repr[:-2]}\n)"
+
+    @classmethod
```

Comment:
@montezdesousa maybe (in the future) this could be something that lives in a `BaseModel` and is inherited for the `CredentialsModel`, `PreferencesModel` etc

Reply:
Yes indeed!